prow/gerrit/adapter/adapter.go

```diff
@@ -545,11 +533,11 @@ func (c *Controller) processChange(logger logrus.FieldLogger, instance string, c
 			optedOut := isProjectOptOutHelp(c.projectsOptOutHelp, instance, change.Project)
 			c.lock.RUnlock()
 			if needsHelp && !optedOut {
-				runWithTestAllNames, optionalJobsCommands, requiredJobsCommands, err := pjutil.AvailablePresubmits(listChangedFiles(change), cloneURI.Host, change.Project, change.Branch, presubmits, logger.WithField("help", true))
+				runWithTestAllNames, optionalJobsCommands, requiredJobsCommands, err := pjutil.AvailablePresubmits(listChangedFiles(change), instance, change.Project, change.Branch, presubmits, logger.WithField("help", true))
```

Comment:
It looks like the org, and repo parameters are not used? Can we just remove them please?

Reply:
Agreed, will do in a separate PR